demands/__init__.py

```
@@ -165,3 +165,10 @@ def pre_send(self, request_params):
             request_params['data'] = json.dumps(
                 request_params['data'], default=str)
         return request_params
+
+    def post_send(self, response, **kwargs):
+        response = super(JSONServiceClient, self).post_send(response, **kwargs)
+        try:
+            return response.json()
+        except ValueError:
+            return None
```

I would expect to get `None` back for a no content response, not a response object.

In the case that the service returns invalid JSON, maybe it's best to let the exception propagate?

I guess I'm not sure what the caller is going to do with the response object since they'll be expecting a decoded JSON object in success case.